doc/development/plugins.rst

```diff
@@ -3,8 +3,8 @@
 
 .. _plugin_overview:
 
-Building a plugin
-=================
+Plugins and operations
```

Comment:
Should I change this back?

Reply:
Perhaps _Building a plugin_ was slightly more introductory as a title, yes

Reply:
Changed it back 🙂